pkg/config/types/bacalhau.go

```diff
@@ -23,7 +24,8 @@ type Bacalhau struct {
 	Logging             Logging             `yaml:"Logging,omitempty"`
 	UpdateConfig        UpdateConfig        `yaml:"UpdateConfig,omitempty"`
 	FeatureFlags        FeatureFlags        `yaml:"FeatureFlags,omitempty"`
-	DisableAnalytics    bool                `yaml:"DisableAnalytics,omitempty"`
+	// DisableAnalytics when set to true disables bacalhau from sharing anonymous user data with Expanso.
```

Comment:
@aronchick @seanmtracey what do you think of this description for the `DisableAnalytics` field? The comment here determines the description text of `bacalhau config list`, so currently this will show:
```
disableanalytics                                  false                        DisableAnalytics when set to true prevents bacalhau from sharing anonymous user  
                                                                                data with Expanso.  
 ```
 
 Is: `DisableAnalytics when set to true prevents bacalhau from sharing anonymous data with Expanso.` the right messaging?

Reply:
"DisableAnalytics when set to true prevents bacalhau from sharing anonymous usage data with the developer team."